tools/dependencies/known-dependencies.txt

```diff
@@ -273,3 +273,14 @@ aws-java-sdk-s3-1.12.160.jar
 aws-java-sdk-kms-1.12.160.jar
 aws-java-sdk-emr-1.12.160.jar
 aws-java-sdk-core-1.12.160.jar
+commons-text-1.8.jar
```

Comment:
You should add those packages to the right place in `dolphinscheduler-dist/release-docs/LICENSE` too

Reply:
> You should add those packages to the right place in `dolphinscheduler-dist/release-docs/LICENSE` too

@zhongjiajie I think I've already added them. Is there anything wrong with the LICENSEs? 
![image](https://user-images.githubusercontent.com/34905992/162354641-b6e667c6-90af-499b-991c-1592a274cd59.png)

Reply:
You should also add them to file https://github.com/apache/dolphinscheduler/blob/81f9b876f9f5865a7e65b668008af0cde580b326/dolphinscheduler-dist/release-docs/LICENSE

Reply:
> You should also add them to file https://github.com/apache/dolphinscheduler/blob/81f9b876f9f5865a7e65b668008af0cde580b326/dolphinscheduler-dist/release-docs/LICENSE

Got it. Thx!

Reply:
> You should also add them to file https://github.com/apache/dolphinscheduler/blob/81f9b876f9f5865a7e65b668008af0cde580b326/dolphinscheduler-dist/release-docs/LICENSE

Fixed in the latest commit.